README.md

```diff
@@ -299,6 +299,10 @@ All Public APIs are documented in our [Include.h](https://github.com/awslabs/ama
 
 Refer to [related](#related) for more about WebRTC and KVS.
 
+## Development
+
+If you would like to contribute to the development of this project, please base your pull requests off of the develop branch, and to the develop branch. Commits from develop will be merged into master periodically as a part of each release cycle.
```

Comment:
Nit: To be clear with the branch name, let us add `origin/develop` or `develop` in code block so that it is clearly visible.